Sources/TuistScaffold/TemplateGenerator.swift

```diff
@@ -81,7 +81,7 @@ public final class TemplateGenerator: TemplateGenerating {
                     renderedContents = fileContents
                 }
             }
-
+            guard !renderedContents.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty else { return }
```

Comment:
Why is this necessary? Should we add a comment?

Reply:
Yeah, I'll add one -> if a file has no characters, we make the assumption that the user did not want to generate this file.
This is then leveraged in some files that are macOS or iOS-only.